doc/examples/ml/neon/README.md

```diff
@@ -0,0 +1,48 @@
+# ML pipeline using Nervana Neon and Pachyderm
+
+![alt tag](pipeline.jpg)
+
+This machine learning pipeline integrates Nervana Neon training and inference into a production scale pipeline using Pachyderm.  In particular, this pipeline trains and utilizes a model that predicts the sentiment of movie reviews, based on data from IMDB.
+
+## Deploying Pachyderm
+
+See the [Pachyderm docs](http://docs.pachyderm.io/en/latest/) for details. Note, this demo requires `pachctl` 1.4.0+.  
+
+## Deploying the pipeline
+
+1. Create the necessary data "repositories":
+
+    ```sh
+    $ pachctl create-repo training
+    $ pachctl create-repo reviews
+    ```
+
+2. Create the pipeline:
+
+    ```sh
+    $ pachctl create-pipeline -f pipeline.json
+    ```
+
+## Running model training
+
+Because we have already deployed the pipeline, the training portion of the pipeline will run as soon as data is committed to the training data repo.  The training data in TSV format can be obtained [here](https://www.kaggle.com/c/word2vec-nlp-tutorial/data).
+
+```sh
+$ pachctl put-file training master labeledTrainData.tsv -c -f labeledTrainData.tsv
+```
+
```

Comment:
Can we give some indication on how long this should take to complete? minutes, hours? We should give users some very rough estimate.